spark-connector/src/main/java/com/datastrato/gravitino/spark/catalog/GravitinoCatalog.java

```diff
@@ -88,32 +157,45 @@ public Table createTable(
 
   @Override
   public Table alterTable(Identifier ident, TableChange... changes) throws NoSuchTableException {
-    throw new NotSupportedException("Doesn't support altering table");
+    throw new NotSupportedException("Doesn't support altering table for now");
   }
 
   @Override
   public boolean dropTable(Identifier ident) {
-    throw new NotSupportedException("Doesn't support drop table");
+    checkNotPathIdentifier(ident, "dropTable");
+    return gravitinoCatalogClient
+        .asTableCatalog()
+        .dropTable(NameIdentifier.of(metalakeName, catalogName, getDatabase(ident), ident.name()));
   }
 
   @Override
-  public void renameTable(Identifier oldIdent, Identifier newIdent)
-      throws NoSuchTableException, TableAlreadyExistsException {
-    throw new NotSupportedException("Doesn't support renaming table");
+  public boolean purgeTable(Identifier ident) {
+    checkNotPathIdentifier(ident, "purgeTable");
+    return gravitinoCatalogClient
+        .asTableCatalog()
+        .purgeTable(NameIdentifier.of(metalakeName, catalogName, getDatabase(ident), ident.name()));
```

Comment:
Missing `Exception` Handling, the drop table too

Reply:
Should I process `RuntimeException` explicitly?

Reply:
You can refer to the exception handling in a module similar to Spark.

Reply:
I will check it latter

Reply:
seems no need to handle exception like trino, `dropTable` return false means table not exists, Spark will check the exists of the table before `dropTable`
```
    boolean dropped =
        tableCatalog.dropTable(
            NameIdentifier.ofTable(
                metalake.name(), catalogName, tableName.getSchemaName(), tableName.getTableName()));
    if (!dropped) {
      throw new TrinoException(GRAVITINO_OPERATION_FAILED, "Failed to drop table " + tableName);
    }
```